app/assets/javascripts/govuk-component/govspeak-enhance-youtube-video-links.js

```diff
@@ -40,6 +40,7 @@
     }
 
     function enhanceYoutubeVideoLinks($el) {
+      if ($($el).hasClass('disable-youtube')) return ;
       $el.find("a[href*='youtube.com'], a[href*='youtu.be']").each(function (i){
```

Comment:
I think it'd be better to not run the script by excluding here:
https://github.com/alphagov/static/blob/4c4dd384e4e92e0e13c508c2c211f8d089125922/app/assets/javascripts/govuk-component/govspeak-enhance-youtube-video-links.js#L77

Should also update the JS specs for the script.